src/applications/personalization/profile/util/index.js

```diff
@@ -102,98 +110,113 @@ const hasErrorMessage = (errors, errorKey, errorText) => {
   );
 };
 
+export const hasErrorCombos = ({
+  errors,
+  errorKeys = [],
+  errorTexts = [],
+} = {}) => {
+  return errorKeys.some(errorKey => {
+    if (errorTexts.length > 0) {
+      return errorTexts.some(errorText =>
+        hasErrorMessage(errors, errorKey, errorText),
+      );
+    }
+    return hasErrorMessage(errors, errorKey);
+  });
+};
+
```

Comment:
I thought this was a good abstraction to remove most of the || that are needed in the below functions, but 🤷🏼. I would be curious if yall had any other ideas for doing combined error checking liek this.